### PR TITLE
Pretty icons and tooltips

### PR DESCRIPTION
**Pull request by JKGisMe**
Let me know any thoughts or suggestions since I kind of just went off on my own here.

I also deleted `application.hbs`. It didn't see to be being used(in favor of the pods style `application/template.hbs`)  but maybe there was something I didn't understand?

### CODE REVIEW

**Comment by rwjblue** [comment]
Can you attach before/after screenshots?

**Comment by JKGisMe** [comment]
Yeah I'll fix the tests and make some ss.

**Comment by JKGisMe** [comment]
Side Note: I can't figure out how to run the tests locally. I get 37 fails locally.

All my changes are in peach-ish bar. The close, add a pane, show/hide file tree, and collapse/expand icons. The tool tips for these as well. I changed the expand/collapse icons because they suggest a maximizing/minimizing of the file tree pane rather than expanding/collapsing the folders within. The old tooltips also took quite a while to appear and could potentially be overlooked. 

### Current
twiddleold

### This PR

twiddlenew

**Comment by stefanpenner** [comment]
fancy!

File: ember-cli-build.js [hunk 1]
```diff
@@ -45,8 +45,15 @@ module.exports = function() {
   });
 
   app.import('bower_components/ember/ember-template-compiler.js');
+  app.import('vendor/hint.min.css');
```
Comment (Gaurav0) [inline ember-cli-build.js]: Please import the unminified version. Ember CLI will minify in production.

Comment (JKGisMe) [inline ember-cli-build.js]: I will need to compile this. I'll see if I still have the files or make a new one.

**Pull request merged**